Source/Scene/Camera.js

```diff
@@ -233,6 +237,12 @@ define([
         mag += mag * Camera.DEFAULT_VIEW_FACTOR;
         Cartesian3.normalize(this.position, this.position);
         Cartesian3.multiplyByScalar(this.position, mag, this.position);
+
+        /**
+         * If set, the camera's position is constrained to be within this bounding object.
```

Comment:
This doc isn't quite right.  It should be like `Gets and sets the bounding object used to constrain the camera's position`

Reply:
I'm not really sure I understand why this needs to be changed? it's not a getter or a setter itself?

Reply:
This is just the standard way we document properties like this. It lets the user know that they can read and write to this property.